src/offset/local/unix.rs

```diff
@@ -91,8 +97,9 @@ struct Cache {
 
 impl Default for Cache {
     fn default() -> Cache {
+        // default to UTC if no local timezone can be found
         Cache {
-            zone: TimeZone::local().expect("unable to parse localtime info"),
+            zone: TimeZone::local().unwrap_or_else(|_| TimeZone::utc()),
```

Comment:
I think this can be written as `unwrap_or_else(TimeZone::utc)`?

Reply:
this should be sorted but the changes don't show up here